### DISCUSSION TITLE
The sly-mrepl-shortcut on the coma ',' -- Is it really good idea?

### DISCUSSION THREAD

**Discussion by hww** [joaotavora/sly#522] [Ideas]
Hi, I found the key binding for `sly-mrepl-shortcut` by default is on `,` In the documentation next text

> sly-mrepl-shortcut
> The key to use to trigger the REPL’s “comma shortcut”. We recommend you keep the default setting which is the comma (,) key, since there’s special logic in the REPL to discern if you’re typing a comma inside a backquoted list or not.

Is it really good idea? The characters ` ' @ and , needed to type the macro body in the REPL. Is it not better to use a longer sequence for by default?

**Comment by joaotavora** [comment]
It has been used in SLIME for almost 20 years and in SLY for more than 8.  So yes: I'd say it's a good idea that we don't want to change the defaults of. There is special code to understand when `,` is in a backquote expression.

**Reply by hww** [reply]
Ok. Got You.
